lib/editor-client.js

```diff
@@ -178,8 +178,10 @@ firepad.EditorClient = (function () {
   }
 
   EditorClient.prototype.onBlur = function () {
-    this.cursor = null
-    this.sendCursor(null)
+    if (this.cursor != null) {
+      this.cursor = null
+      this.sendCursor(null)
+    }
     this.focused = false
```

Comment:
Why didn't you rely on `focused` tho?

Reply:
2 reasons:
 - seems more legit to chose not to send the cursor by checking if it has not changed
 - it doesn't work because the focus event is triggered before

Reply:
ok :+1: